src/protocol/context.rs

```diff
@@ -55,7 +55,7 @@ impl<'a, N> ProtocolContext<'a, N> {
 
     /// Get the PRSS instance for this step.
     #[must_use]
-    pub fn prss(&self) -> &PrssSpace {
+    pub fn prss(&self) -> &SharedRandomness {
```

Comment:
do you plan to rename method name as well?

Reply:
Hmm, what we're really looking at is IndexedSharedRandomness (as opposed to SequentialSharedRandomness for the rand::Rng implementation).  Maybe I should revisit this again.